packages/components/tree-v2/src/tree.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+  <div
+    ref="el$"
+    class="el-tree"
+    :class="{
+      'el-tree--highlight-current': highlightCurrent,
+    }"
+    role="tree"
+  >
+    <fixed-size-list
+      v-if="isNotEmpty"
+      class-name="el-tree-virtual-list"
+      :data="flattenTree"
+      :total="flattenTree.length"
+      width="100%"
+      :height="height"
+      :item-size="26"
```

Comment:
Maybe we can expose the `item-size` for users since they might have special needs.

Reply:
> Maybe we can expose the `item-size` for users since they might have special needs.

The item-size of tree.scss is hardcode, maybe the virtual tree needs a separate style file?

Reply:
This feature may be supported in the future